.github/workflows/developer-portal-healthcheck.yml

```diff
@@ -0,0 +1,19 @@
+# This workflow triggers a Healthcheck workflow in developer-portal for every PR targeting main branch.
+name: Healthcheck
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  create-healthcheck:
```

Comment:
I would rename this workflow and job to something that tells me this job creates a preview build. I understand a healthcheck as an Rest endpoint that you ping and you get a response back that the system is running without any issues.